renderdoc/driver/d3d12/d3d12_pixelhistory.cpp

```diff
@@ -1128,7 +1135,7 @@ struct D3D12ColorAndStencilCallback : public D3D12PixelHistoryCallback
       D3D12CopyPixelParams depthCopyParams = targetCopyParams;
       depthCopyParams.srcImage = depthImage;
       depthCopyParams.srcImageFormat = GetDepthSRVFormat(depthImage->GetDesc().Format, 0);
-      depthCopyParams.copyFormat = DXGI_FORMAT_R32_TYPELESS;
+      depthCopyParams.copyFormat = GetDepthCopyFormat(depthFormat);
```

Comment:
Should this TODO be removed now

      // TODO: What about D16? Still copy as 32 bit?

Reply:
Thanks, I missed that.